subsys/bluetooth/host/conn.c

```diff
@@ -1178,7 +1178,7 @@ void bt_conn_set_state(struct bt_conn *conn, bt_conn_state_t state)
 
 			bt_conn_unref(conn);
 			break;
-		case BT_CONN_CONNECTING_SCAN:
+		case BT_CONN_CENTRAL_SCAN_BEFORE_CONNECTING:
 			/* this indicate LE Create Connection with peer address
```

Comment:
This seems to indicate that in that state, we're not scanning, we're initiating directly.

Reply:
From the applications point of view we are initiating directly - but the host first started scanning. Maybe we should rephrase this comment to:
```suggestion
			/* this indicate connection establishment with peer address
```

Reply:
yes that sounds less misleading
edit: maybe at some point it was indeed HCI LE Create Connection, but the code evolved...